CondTools/SiStrip/plugins/SiStripNoisesFromDBMiscalibrator.cc

```diff
@@ -253,15 +263,17 @@ void SiStripNoisesFromDBMiscalibrator::analyze(const edm::Event& iEvent, const e
   // write out the SiStripNoises record
   edm::Service<cond::service::PoolDBOutputService> poolDbService;
 
-  if (poolDbService.isAvailable())
-    poolDbService->writeOneIOV(*theSiStripNoises, poolDbService->currentTime(), "SiStripNoisesRcd");
-  else
+  if (poolDbService.isAvailable()) {
+    if (poolDbService->isNewTagRequest("SiStripNoisesRcd")) {
+      poolDbService->createOneIOV(theSiStripNoises, poolDbService->currentTime(), "SiStripNoisesRcd");
+    } else {
+      poolDbService->appendOneIOV(theSiStripNoises, poolDbService->currentTime(), "SiStripNoisesRcd");
+    }
+  } else {
```

Comment:
Just for my understanding: why do you split `writeOneIOV` back into `createOneIOV+appendOneIOV`?
IIUC if the second paramenter (the IOV) is the same, then `writeOneIOV` can be used.

Reply:
I like to have it explicit.